src/store/token.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+export async function getToken(token) {
+	const tokenValue = await ((typeof (token) === 'function')
+		? token()
+		: token);
+
+	return await new Token(tokenValue);
+}
+
+export function compareTokens(a, b) {
+	return a.toString() === b.toString();
+}
+
+export const TOKEN_COOKIE = -1;
+export const TOKEN_COOKIE_CACHE_KEY = 'cookie';
```

Comment:
How are these related to `Token` class?

Reply:
If there is no token then the token will be sent through a cookie. So it sets the state of token to cookies deal with it.

Reply:
Do these need to be exported or are they only used within the `Token` class?

Reply:
Good point, I don't think they do need to be exported.